### DISCUSSION TITLE
TIP Statement Of Compliance

### DISCUSSION THREAD

**Discussion by Radula76** [magma/magma#11672] [Q&A] [answered]
Hi team,
  We were wondering if someone had done a statement of compliance towards TIP OCN requirements:
https://telecominfraproject.com/open-core-network/
https://cdn.brandfolder.io/D8DI15S7/at/prcqhzp3m6z8v786gcrk6bf/TIP_OCN_FWA_TECH_REQ_DOC_version_1_FINAL_REDdocx.pdf

here is my initial take on the compliance, I haven't gone into detail on every feature currently in 1.6, I'm working through that. I hoped that someone with more knowledge on each area can assist in some text for the relevant sections?
Our goal isn't to meet the TIP standards, we just want a benchmark where we currently are and what the main missing components or strategies are.

Thanks for your support.

Mark - Radtonics.
[Magma 1.6 Compliance to TIP OCN FWA Rev1.0 Dec 16_2021.xlsx](https://github.com/magma/magma/files/8095102/Magma.1.6.Compliance.to.TIP.OCN.FWA.Rev1.0.Dec.16_2021.xlsx)

**Comment by ulaskozat** [comment, marked as answer]
Please connect with Xinli Hou as these requirements were cross-checked and commented in an email thread with OCN.
